### ISSUE TITLE
maxtime=16384 ? Units?

### ISSUE THREAD

**Issue by apiszcz** [dpwe/audfprint#8]
Does -- maxtime parameter represent the number of samples or seconds?

**Comment by dpwe** [comment]
It's frames, which are 256/sr, or ~23 ms at 11 kHz. Thus, the time alias
window is about 6 minutes by default; durations beyond this can still be
matched, but the reported offsets will be modulo 380 seconds.

  DAn.

**Comment by apiszcz** [comment]
​Thank you, and another easy question. At 8KHZ my ​maxtime will be
.032*16384 or 524.288 seconds. That works out good for my case.

Since all my data is 8KHZ mono, do I need to resample to 8KHZ? or am 'OK' I
without specifying the --resample flag 8000?, I noted the resample option
mentions a default of 11KHZ.

Thank you.

**Comment by dpwe** [comment]
You should specify --resample 8000 to *prevent* the data from being
resampled.  Yeah, maybe not the most intuitive option name.

  DAn.

**Comment by apiszcz** [comment]
Thanks, I really like all the parameters you expose, now to understand them
with my use case.
I need to test -I and more of the capabilities.​ I'm interested in any
tidbits like: http://www.ee.columbia.edu/~dpwe/resources/matlab/audfprint/

**Comment by dpwe** [comment]
To explain: This Python codebase has the same name as the Matlab code you link to, and in fact the Python code will attempt to read databases written by Matlab, and they may work.  I did the original implementation in Matlab and spent several years enhancing that version, but then I chose to start from scratch with a Python version to make it smaller, cleaner, and easier for everyone to use.  Only the core functionality was supported in the original Python version, although I'm gradually adding things.  The reporting of time support that I recently added in Python was something I never got around to in the Matlab.
